app/frontend/src/store/tasks.ts

```diff
@@ -0,0 +1,44 @@
+import { createEffect, createStore } from 'effector';
+import { createTask, getTasks, updateTask } from 'services/tasks';
+import { Task } from 'types/entities';
+
+
+/**
+ * Tasks store
+ */
+export const $tasks = createStore<Task[]>([]);
+
+/**
+ * Task effects
+ */
+export const getTasksEffectFx = createEffect(getTasks);
```

Comment:
what does the `Fx` mean?

Reply:
it's naming convention
https://github.com/effector/eslint-plugin/blob/master/rules/enforce-effect-naming-convention/enforce-effect-naming-convention.md

Reply:
Yes, but `Fx` already means `Effect`, so the `getTasksEffectFx` contains the "effect" word twice.

In the example you show, there is no `effect` word, just 'Fx' suffix.

Maybe `getTasksFx` would be a better name?

Reply:
good point, renamed